tests/remote_store/remote_store/restore.yaml

```diff
@@ -0,0 +1,26 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+
+description: Test restore from a remote store.
+prologues:
+  - path: /_bulk
+    method: POST
+    parameters:
+      refresh: true
+    request:
+      content_type: application/x-ndjson
+      payload:
+        - {create: {_index: books, _id: book1}}
+        - {author: Harper Lee, title: To Kill a Mockingbird, year: 60}
```

Comment:
Sorry to being annoying, please make it 1960 and i can approve this PR

Reply:
done